<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="NDI-LAP-645" failures="2" tests="2" name="Login" time="10.739" errors="0" timestamp="08 Feb 2022 21:37:12 GMT">
  <testcase classname="finalAssesment1.LoginCart" name="Login" time="0.009">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.aventstack.extentreports.ExtentReports.createTest(String)&amp;quot; because &amp;quot;this.extent&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentReports.createTest(String)" because "this.extent" is null
at CommonLib.Listeners.onTestStart(Listeners.java:24)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- Login -->
  <testcase classname="finalAssesment1.LoginCart" name="Login" time="10.722">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.aventstack.extentreports.ExtentTest.fail(java.lang.Throwable)&amp;quot; because &amp;quot;this.test&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.fail(java.lang.Throwable)" because "this.test" is null
at CommonLib.Listeners.onTestFailure(Listeners.java:38)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- Login -->
</testsuite> <!-- Login -->
